sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -51,6 +51,93 @@ APIs for managing module identities, module twins, and querying modules
 
 ```csharp
 
+public  class Modules
+{
+    /// <summary>
+    /// Create a device module.
+    /// </summary>
+    /// <param name="moduleIdentity">The module to create.</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns></returns>
+    public virtual async Task<Response<ModuleIdentity>> CreateIdentityAsync(ModuleIdentity moduleIdentity, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Get a single device module.
```

Comment:
"Get a single module identity" to clarify that this doesn't include twin. No need to mention the device here, either